core/builtins/src/kotlin/Progressions.kt

```diff
@@ -98,7 +103,12 @@ public open class IntProgression
 
     override fun iterator(): IntIterator = IntProgressionIterator(first, last, step)
 
-    /** Checks if the progression is empty. */
+    /**
+     * Checks if the progression is empty.
+     *
+     * Increasing progression is empty if its start value is greater than the end value.
+     * Decreasing progression is empty if its start value is less than the end value.
```

Comment:
A progression doesn't have start and end values. It has the properties returning `first` and `last` elements. The difference is explained here: https://kotlinlang.org/docs/ranges.html#progression

Reply:
Changed the wording to `start`/`end` elements